Dockerfile

```
@@ -36,6 +34,24 @@ COPY --from=voila_rootfs_builder /etc/ /etc/
 
 
 FROM base_image_python AS kernel_rootfs_builder
+RUN apt-get update && apt-get install -y --no-install-recommends wget
+# Miniconda
+ARG conda_dir="/usr/miniconda3"
+ENV PATH="${conda_dir}/bin:${PATH}"
+ARG PATH="${conda_dir}/bin:${PATH}"
+ARG mconda="Miniconda3-py37_4.12.0-Linux-x86_64.sh"
```

Currently we use python 3.8 for Voila, so I think this should be
```suggestion
ARG mconda="Miniconda3-py38_4.12.0-Linux-x86_64.sh"
```